scripts/dynamo/set-maintenance-mode.ts

```diff
@@ -0,0 +1,68 @@
+import { DynamoDBClient } from '@aws-sdk/client-dynamodb';
+import { DynamoDBDocument } from '@aws-sdk/lib-dynamodb';
+import { InvokeCommand, LambdaClient } from '@aws-sdk/client-lambda';
+
+// # Usage
+// #   npx ts-node set-maintenance-mode.ts true dev
+
+// # Arguments
+// #   - $1 - true to engage maintenance mode, false to disengage maintenance mode
+// #   - $2 - the environment to set the flag
```

Comment:
Could use the ENV environment variable here since we're already assuming the presence of the AWS environment variables for the Dynamo calls.